doc/02-Installation.md.d/From-Source.md

```diff
@@ -0,0 +1,22 @@
+# Installing Icinga Notifications Web from Source
+
+Please see the Icinga Web documentation on
+[how to install modules](https://icinga.com/docs/icinga-web-2/latest/doc/08-Modules/#installation) from source.
+Make sure you use `notifications` as the module name. The following requirements must also be met.
+
+### Requirements
+
+- PHP (≥7.2)
+- PHP needs the following extensions to be installed and activated:
+    - `cURL`
+    - `dom`
+    - `json`
+    - `libxml`
+- [MySQL](https://www.php.net/manual/en/ref.pdo-mysql.php)
+  or [PostgreSQL](https://www.php.net/manual/en/ref.pdo-pgsql.php) PDO PHP libraries
+- [Icinga Notifications Daemon](https://github.com/Icinga/icinga-notifications)
+- [Icinga Web 2](https://github.com/Icinga/icingaweb2) (≥2.9)
```

Comment:
`Icinga Web` is the product name nowadays

Reply:
They use `daemon` to differentiate between the golang and web part over at https://github.com/Icinga/icinga-notifications/pull/210.
I do agree that it might seem superflous, but just keeping it `Icinga Notifications` and `Icinga Notifications Web` might seem more confusing to and enduser.

Reply:
We might want to update this in the other documentations as well.
Fixed it here.